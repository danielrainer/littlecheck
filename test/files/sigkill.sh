# RUN: /bin/sh %s
kill -KILL $$
