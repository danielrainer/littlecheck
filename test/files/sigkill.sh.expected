Failure in sigkill.sh:

  Process was unexpectedly killed by signal SIGKILL (Killed)

  when running command:
    /bin/sh sigkill.sh
